source/D2Game/src/GAME/CCmd.cpp

```diff
@@ -195,7 +195,7 @@ void __fastcall CCMD_ProcessClientSystemMessage(void* pData, int32_t nSize)
     {
         if (sub_6FC38100(nClientId))
         {
-            GAME_DisconnectClientById(nClientId, 0);
+            GAME_DisconnectClientById(nClientId, EVENTTYPE_DROPTIMEOUT);
```

Comment:
Should we rename this enum value ? It seems to be used as `EVENTTYPE_DISCONNECT` rather than only for timeout.